ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-ref.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-ref.yaml",
```

Comment:
we are supposed to create a dedicated subfolder in `ibmcloud` for the check point.

Reply:
cucushift-installer-check-ibmcloud-commands.sh is for ibmcloud normal checking, cucushift-installer-check-ibmcloud-dedicatedhost-commands.sh is for dedicated host checking.

Reply:
ok.